### DISCUSSION TITLE
Assessment of the difficulty in porting CPU architecture for the moosefs

### DISCUSSION THREAD

**Discussion by wangyuliu** [moosefs/moosefs#548] [Ideas]
Hello everyone! I am working on implementing a tool to assess the complexity of CPU architecture porting. It primarily focuses on RISC-V architecture porting. As part of my dataset, I have collected the MooseFs project. **I would like to gather community opinions to support my assessment.** I appreciate your help and response! Based on scanning tools, the porting complexity is determined to be simple, with a small amount of code related to the CPU architecture in the project. (referring to the overall workload from adapting the project to a specific architecture to achieving full functionality on that architecture). Is this assessment accurate? I look forward to your help and response.
各位好！我在实现一个关于CPU架构的移植工作复杂度评估的工具，主要面向riscv架构移植，我收集MooseFs项目作为我的数据集之一，需要收集社区的意见作为支撑，期待您的帮助和回答！通过扫描工具得出其移植复杂度为简单，工程中与CPU架构相关的代码量也较少（这里指的是从工程尚未适配某架构到实现某架构的较完整功能的总体工作量）这样的判断是否准确？期待您的帮助与回答

**Comment by tianon** [comment]
As discussed in the duplicate https://github.com/moosefs/moosefs/discussions/549, this is already supported.
